server/server.js

```diff
@@ -340,10 +354,20 @@ app.ws('/init', function (ws, req) {
 });
 
 // Server
-app.all('/*', (req, res, next) => { // needed for react router
-  res.sendFile(__dirname + '/build/index.html');
+
+const template = path.resolve(__dirname + "/../build/index.html");
```

Comment:
This resolves an artifact from the folder restructure you had done previously. Express doesn't allow the use of directory traversal, so need to use `path.resolve` before inlining the string

Reply:
Ok i get it, I think I've had this issue trying to move my server file!